scripts/openstack.py

```diff
@@ -187,6 +187,11 @@ def get_suite_parser():
         help=("Use tasks and suite definition in this repository"),
         default=os.getenv('TEUTH_SUITE_REPO', 'https://github.com/ceph/ceph'),
     )
+    parser.add_argument(
+        '--sleep-before-teardown',
+        help='Number of seconds to sleep before tearing down the target VMs',
```

Comment:
s/before tearing down ../before the teardown of first task in the job/, 

I think we should really add the warning to limit on how many jobs to be run when using this option, As it can hold quite a bit of hardware if not used properly. A warning when the CLI is invoked is needed.

Reply:
Hm.... not sure I understood the difference from English point of view, but this made me to turn attention to 'target VMs' which is definitely must be change to something appropriate.

Reply:
@smithfarm I wonder, if should rename the feature to something like --sleep-before-unwinding-tasks (confused). I believe original feature name was call after teuthology-openstack kung-fu. It is okay to me still have a sleep-before-teardown since there is no teardown operation defined in the native teuthology, would this be obvious what does the 'teardown' mean to others?

Reply:
`--sleep-before-unwind` would be shorter, but isn't this splitting hairs?

To put it another way, what is wrong with "sleep before teardown"? Is the sleep not kicking in before teardown here? (There are only three possibilities: before, at the same time as, or after.)

Reply:
I'm okay with 'teardown', nevertheless there is no definition of _teardown_ in teuthology, just wondering, since it is not clear what teardown does it mean, if it means target unlock, then it is not right before teardown. As for teuthology-openstack teardown also means teuthology server teardown.

Reply:
> there is no definition of teardown in teuthology

This is not a true statement. Teuthology tasks written in the "new style" (using a class inheriting from `Task`)  have a "teardown" phase: https://github.com/ceph/teuthology/blob/master/teuthology/task/__init__.py#L111-L117

Let me repeat: I don't think we should split hairs here. The name "sleep before teardown" is sufficiently clear and easy to understand.